website/docs/r/cluster.html.markdown

```diff
@@ -349,6 +349,7 @@ But in order to explicitly change `provider_instance_size_name` comment the `lif
 * `provider_auto_scaling_compute_max_instance_size` - (Optional) Maximum instance size to which your cluster can automatically scale (e.g., M40). Required if `autoScaling.compute.enabled` is `true`.
 
 * `replication_specs` - Configuration for cluster regions.  See [Replication Spec](#replication-spec) below for more details.
+* `paused` (Optional) - Flag that indicates whether the cluster is paused or not. **NOTE** If `paused` is set `true`, it will only last 30 days, after 30 days, it will unpause automatically, to avoid further changes related to `paused` parameter please ignore the parameter with the lifecycle or re-apply to pause again.
```

Comment:
Understood, for additional work of advanced clusters, should I push in this same PR?

Reply:
Added changes for advanced cluster, indeed, that you can't pause with a tenant cluster, it will appear an error about `Cannot update a M0/M2/M5 cluster through the public API`

Reply:
Well that's not a great error message but that's the API. Much to work on there!  Thanks Edgar.